scripts/src/middleware.ts

```diff
@@ -116,3 +117,16 @@ function serverErrorHandler(err: any, req: Request, res: Response) {
 	logger.error(message);
 	res.status(500).send({ code: 500, error: err.message || "Server error", message:  err.message });
 }
+
+export const statusHandler = async function(req: express.Request, res: express.Response) {
+	res.status(200).send(
+		{
+			status: "ok",
+			app_name: process.env.APP_NAME,
```

Comment:
put this in the config.
everything that comes form the `process.env` should go into the config